kooplex-proxy/proxy/crud.py

```diff
@@ -1,33 +1,74 @@
 from sqlalchemy.orm import Session
-from sqlalchemy import text
+from sqlalchemy.ext.declarative import DeclarativeMeta
 
 import models
-from database import engine
+from database import SCHEMA_SET
```

Comment:
As I asked in the in the previous PR, please change the data type to dictionary, where the keys are the nicknames of the schemas. Please discuss with Krizstian to agree on the keys.